docker/README.md

```diff
@@ -0,0 +1,29 @@
+# Ed-Fi-Analytics-Middle-Tier docker local configuration
+
+This docker container satisfies the user story:
+> As an AMT developer, I want to run integration tests using PostgreSQL and SQL Server in Docker containers on my localhost.
```

Comment:
I like to wrap long lines in markdown files at around 80 characters so that it is easier to read the plain text. The extension `stkb.rewrap` in VS Code is great for this.

```suggestion
> As an AMT developer, I want to run integration tests using PostgreSQL and SQL
> Server in Docker containers on my localhost.
```